modules/openy_features/openy_node/config/install/core.entity_view_mode.node.facility.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
Let's be more generic. Instead of `facility`, this view mode can be called "Sedebar teaser", so it will be applicable to all nodes.